packages/stratimikos/adapters/belos/src/Thyra_BelosLinearOpWithSolveFactory_def.hpp

```diff
@@ -77,28 +68,10 @@ namespace Thyra {
 template<class Scalar>
 const std::string BelosLinearOpWithSolveFactory<Scalar>::SolverType_name = "Solver Type";
 template<class Scalar>
-const std::string BelosLinearOpWithSolveFactory<Scalar>::SolverType_default = "Pseudo Block GMRES";
+const std::string BelosLinearOpWithSolveFactory<Scalar>::SolverType_default = "PSEUDO BLOCK GMRES";
```

Comment:
Does this actually matter?  I don't mind the change, but I think `Belos::SolverFactory` capitalizes and otherwise canonical-izes the input name.  Also, this is not consistent with a default set below.

Reply:
This probably does not matter. Fixed the issue with different defaults given below.